### ISSUE TITLE
Evoland lacks music (needs MessageChannel support)

### ISSUE THREAD

**Issue by Firlaev-Hans** [ruffle-rs/ruffle#14493]
### Describe the bug

As already described in the more general Evoland issues, both the [Linux](https://github.com/ruffle-rs/ruffle/issues/13426) and [Windows](https://github.com/ruffle-rs/ruffle/issues/13436) versions of the game have sound effects but no music whatsoever, likely because Ruffle lacks support for the `MessageChannel` class.

I've been asked to open a new issue for this.

### Expected behavior

Music should be present in the main menu as soon as you start the game, as well as in-game once unlocked after a few minutes of gameplay.

### Content Location

The game can't be redistributed, but some of the developers already have the files.

### Affected platform

Desktop app

### Operating system

Fedora Linux 39 / KDE Plasma Wayland

**Comment by Aaron1011** [comment]
I don't think this is caused by missing `MessageChannel` support - there's a fallback path for when `Worker` is not supported. This needs support for dynamically generated sounds: https://github.com/ruffle-rs/ruffle/pull/14203
